packages/salesforcedx-vscode-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  DebugSession,
+  InitializedEvent,
+  OutputEvent
+} from 'vscode-debugadapter';
+import { DebugProtocol } from 'vscode-debugprotocol';
+import { SessionService } from '../core/sessionService';
+import { nls } from '../messages';
+import { CommandOutput } from '../utils/commandOutput';
+
+export interface LaunchRequestArguments
+  extends DebugProtocol.LaunchRequestArguments {
+  userIdFilter: string;
```

Comment:
Are all of these required? 
If some are not required, you could mark them as `userIdFilter?: string`.